gen/docdb

```diff
@@ -36,11 +36,11 @@ my $outputData = {
 						 members => [],
 					 };
 
-foreach my $resourceData ($data->getChildElements) {
-	processResourceData $resourceData;
+foreach my $resourceId ($data->getResourceIds()) {
```

Comment:
`our $A_RESOURCE_VO_NAME;` is no longer used and that line can be removed.

Reply:
removed